DataFormats/TrackingRecHit/src/classes_def.xml

```diff
@@ -31,7 +31,7 @@
   <class name="edm::ClonePolicy<TrackingRecHit>"/> <!-- Root6 -->
   <class name="edm::Ref<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit,edm::refhelper::FindUsingAdvance<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit> >"/>
   <class name="edm::RefVectorIterator<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit,edm::refhelper::FindUsingAdvance<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit> >"/>
-  <class pattern="std::iterator<*edm::Ref*>"/>
+  <!--<class pattern="std::iterator<*edm::Ref*>"/>-->
```

Comment:
please add a comment inline to clarify why it's commented out.

@Dr15Jones are the ```std::iterator```s actually usable? I suppose the iterators are not streamed and this is here for iteration in FWLite